crates/bundler/Cargo.toml

```diff
@@ -17,3 +17,6 @@ aa-bundler-primitives = { path = "../primitives" }
 anyhow = "1"
 ethers = { workspace = true }
 tracing = { workspace = true }
+ethers-flashbots = "0.13.1"
```

Comment:
sort deps

Reply:
what are the formatting and sorting rules for Cargo.toml

Even if I sorted the deps alphabetically, it when run `make lint` it still says not sorted

Reply:
Hmm, it should be alphabetically and nothing else. Is there any error text?

Reply:
```
error: Cargo.toml for bundler is not formatted
Checking bundler...
error: Dependencies for bundler are not sorted
Checking contracts...
Checking grpc...
Checking primitives...
Checking rpc...
Checking uopool...
Checking examples...
error: Cargo.toml for examples is not formatted
```

If you check the `example/Cargo.tomal`, and `bin/bundler/Cargo.toml` in the latest commit, they should be alphabetically sorted.

Reply:
It's not sorted because tokio should be before tracing